components/MashCards/MainCard.tsx

```diff
@@ -0,0 +1,80 @@
+import { MashReferral } from 'types';
+import ContactCard from './ContactCard';
+import InitialDecisionCard from './InitialDecisionCard';
+import ScreeningCard from './ScreeningCard';
+import FinalDecisionCard from './FinalDecisionCard';
+
+interface Props {
+  filter: string;
+  mashReferrals: MashReferral | any;
+}
+
+const MainCard = ({ filter, mashReferrals }: Props): React.ReactElement => {
```

Comment:
Just need to write some tests for this.
I'm happy to take this on.
Can do as a future PR.

Reply:
Sounds good!